web/apps/labelstudio/src/pages/CreateProject/Import/Import.jsx

```diff
@@ -73,7 +75,7 @@ function getFiles(files) {
 
 const Footer = () => {
   return (
-    <Modal.Footer>
+    <Modal.Footer style={{ display: "flex", alignItems: "center", justifyContent: "center", gap: "0.5rem" }}>
```

Comment:
```
    <Modal.Footer style={{ display: "flex", alignItems: "center", justifyContent: "center", gap: "0.5rem" }}>
```
Why not encode this into the Footer styles directly?

Reply:
yes i suspect this is a specific situation but i do like the tailwind class approach, +1 to this suggestion